app/serializers/api/v3/columns/column_serializer.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V3
+    module Columns
+      class ColumnSerializer < ActiveModel::Serializer
+        attributes :id, :name, :position, :group, :is_default, :is_geo
+
+        attribute :profile_type do
```

Comment:
@santostiago you can join with the "profiles" table to load this from the DB, and not have it hardcoded

Reply:
That's true, I forgot about that - sorry @santostiago